modules/manage/pages/iceberg/choose-iceberg-mode.adoc

```diff
@@ -0,0 +1,151 @@
+
+ifndef::env-cloud[]
+In the Schema Registry wire format, a "magic byte" and schema ID are embedded in the message payload header. Producers to the topic must use the wire format in the serialization process so Redpanda can determine the schema used for each record, use the schema to define the Iceberg table, and store the topic values in the corresponding table columns. See also: xref:{schema-id-val-doc}[] and 
+the https://www.redpanda.com/blog/schema-registry-kafka-streaming#how-does-serialization-work-with-schema-registry-in-kafka[Understanding Apache Kafka Schema Registry^] blog post.
+
+ifdef::env-cloud[]
+In the Schema Registry wire format, a "magic byte" and schema ID are embedded in the message payload header. Producers to the topic must use the wire format in the serialization process so Redpanda can determine the schema used for each record, use the schema to define the Iceberg table, and store the topic values in the corresponding table columns. See also: the https://www.redpanda.com/blog/schema-registry-kafka-streaming#how-does-serialization-work-with-schema-registry-in-kafka[Understanding Apache Kafka Schema Registry^] blog post.
+endif::[]
+
+=== value_schema_latest
+
+Creates an Iceberg table whose structure matches the latest schema registered for the subject in the Schema Registry. You must register a schema in the xref:manage:schema-reg/schema-reg-overview.adoc[Schema Registry]. Unlike the `value_schema_id_prefix` mode,  `value_schema_latest` does not require that producers use the wire format.
+
+=== disabled
+
+Default for `redpanda.iceberg.mode`. Disables writing to an Iceberg table for the topic.
+
+== Configure Iceberg mode for topic
+
+You can set the Iceberg mode for a topic when you create the topic, or you can update the mode for an existing topic.
+
+.Create a new topic and set `redpanda.iceberg.mode`:
+[,bash]
+----
+rpk topic create <topic-name> --topic-config=redpanda.iceberg.mode=<iceberg-mode>
+----
+
+.Set `redpanda.iceberg.mode` for an existing topic:
+[,bash]
+----
+rpk topic alter-config <new-topic-name> --set redpanda.iceberg.mode=<iceberg-mode>
+----
+
+[[override-value-schema-latest-default]]
+=== Override `value_schema_latest` default
+
+In `value_schema_latest` mode, only the string `value_schema_latest` is required in the property value. This sets `value_schema_latest` mode to its default behavior, which derives the subject for the topic using xref:manage:schema-reg/schema-id-validation.adoc#set-subject-name-strategy-per-topic[TopicNameStrategy]. For Protobuf data, the default behavior also deserializes records using the first message within the corresponding Protobuf schema in the Schema Registry.
+
+To override the default behavior, use the following optional syntax:
```

Comment:
Would be nice to have a line or two about use cases where it would require you to override the default--to give users a sense of the contexts in  which this would make sense or be required.

Reply:
Added use case (different subject name strategy)